tests/test_repository_pypi.py

```diff
@@ -6,53 +6,16 @@
 from piptools.repositories.pypi import open_local_or_remote_file
 
 
-@pytest.mark.network
-def test_generate_hashes_all_platforms(from_line, pypi_repository):
+def test_generate_hashes_all_platforms_local_package(
```

Comment:
```suggestion
def test_generate_hashes_all_platforms(
```
I think we could keep the name of the test.

Reply:
Done.